src/Command/CommandContext.php

```diff
@@ -61,23 +67,60 @@ public function iRunWithArrayParameters($name, $params)
     }
 
     /**
+     * @param string $name
+     * @param array  $params
+     * @param string $dialogString
+     */
+    public function iRunDialogConsoleWithArrayParameters($name, array $params, $dialogString = 'Y')
+    {
+        $application = $this->initApplication();
+
+        $applicationTester = new ApplicationTester($application);
+        $applicationTester->run(array());
```

Comment:
Unfortunately yes. Otherwise it won't load declared Command CLI in the application container.
This is the only thing I found in order to get `$application->find($name);` to work.